terraform/environment/withdrawal-shadowfork-1/ethpandops-com.tf

```diff
@@ -0,0 +1,10 @@
+resource "cloudflare_record" "dns-entry" {
+  for_each = {
+    for vm in local.digitalocean_vms : "${vm.id}" => vm
+  }
+  zone_id = data.sops_file.cloudflare.data["zones.ethpandaops-com.zone_id"]
+  name    = "${each.value.name}.${data.sops_file.cloudflare.data["zones.ethpandaops-com.domain"]}"
```

Comment:
We said we gonna stick with .io. Should revert this.

Reply:
yessir, will change